src/app.html

```diff
@@ -2,29 +2,31 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <meta
-            name="description"
-            content="Appwrite is an open-source platform for building applications at any scale, using your preferred programming languages and tools." />
-        <link rel="icon" type="image/svg+xml" href="%sveltekit.assets%/logos/appwrite-icon.svg" />
-        <link rel="mask-icon" type="image/png" href="%sveltekit.assets%/logos/appwrite-icon.png" />
+        <script>
+            const profile = '%sveltekit.env.PUBLIC_CONSOLE_PROFILE%';
+            const isStudio = profile === 'studio';
+            const baseLogosPath = '%sveltekit.assets%/logos';
+            const appleTouchBase = isStudio ? `${baseLogosPath}/imagine` : baseLogosPath;
+            const description = isStudio
+                ? 'Build something real. Turn your ideas into real products by chatting with the most complete AI builder available to date'
+                : 'Appwrite is an open-source platform for building applications at any scale, using your preferred programming languages and tools.';
+            const iconSvg = isStudio
+                ? `${baseLogosPath}/imagine-icon.svg`
+                : `${baseLogosPath}/appwrite-icon.svg`;
+            const iconPng = isStudio
+                ? `${baseLogosPath}/imagine-icon.png`
+                : `${baseLogosPath}/appwrite-icon.png`;
+            const appleTouchSizes = ['180x180', '167x167', '152x152', '120x120'];
```

Comment:
lets try to avoid JS here. we cnn do 

```html
        <link rel="icon" type="image/svg+xml" href="%sveltekit.assets%/logos/%sveltekit.env.PUBLIC_CONSOLE_PROFILE%-icon.svg" />
```

also lets just remove the meta description here and just do it in the first +layout.svelte

Reply:
and in the +layout.svelte we can use svbelte templates conditions `{#if ...}`